MediaBrowser.Controller/IServerApplicationHost.cs

```diff
@@ -83,7 +83,7 @@ public interface IServerApplicationHost : IApplicationHost
 
         void EnableLoopback(string appName);
 
-        WakeOnLanInfo[] GetWakeOnLanInfo();
+        IEnumerable<WakeOnLanInfo> GetWakeOnLanInfo();
```

Comment:
Hmm, this should be removed.

Reply:
Yeah, I was considering that in another pull request. I actually think we should leave the sync and convert endpoints in for now because those may be useful, but I can't envision a version of Jellyfin that benefits from custom WoL handling.

Reply:
In this PR, or another one?

Reply:
I don't really care. Either is fine, but we removed WoL in a lot of places so it's pretty much dead.